LFS171x/fabric-material/tuna-app/README.md

```diff
@@ -3,4 +3,7 @@
 This application demonstrates the creation and transfer of tuna fish shipments between actors leveraging Hyperledger Fabric in the supply chain. In this exercise we will set up the minimum number of nodes required to develop chaincode. It has a single peer and a single organization.
 
 if getting error about running ./startFabric.sh permission 
-chmod a+x startFabric.sh 
\ No newline at end of file
+
+chmod a+x startFabric.sh
+
+This application is based on (https://github.com/hyperledger/fabric-samples). Big thanks to the Hyperledger Team and their [Maintainers](http://hyperledger-fabric.readthedocs.io/en/latest/MAINTAINERS.html).
```

Comment:
Hyperledger Team --> Hyperledger Fabric commumity

Reply:
In addition, instead of contributed, we should use same phrasing that you used in the README. Something like "This code is based on code written by the Hyperledger Fabric community."

Reply:
rephrased